File: fortran/problem_9.f90
program problem_9
    integer a
    integer b
    integer c
    integer total
    total = 0
    do c=100,999
        do b=100,c
            do a=100,b
                if (a + b + c == 1000) then
                    if (a**2 + b**2 == c**2) then
                        print *,"A:", a, "B:", b, "C:", c
                        print *,"Problem 9:", a*b*c
                        stop ! We found the correct solution, no need to keep testing
                    end if
                end if
            end  do
        end  do
    end  do
end program problem_9

File: python/problem_9.py
from __future__ import print_function
stop = False
for c in range(100,1000):
    for b in range(100,c):
        for a in range(100,b):
            if a + b + c == 1000:
                if a**2 + b**2 == c**2:
                    print("A:", a, "B:", b, "C:", c)
                    print("Problem 9:", a*b*c)
                    stop = True
            if stop: break
        if stop: break
    if stop: break